test/jdk/javax/swing/JFileChooser/8080628/bug8080628.java

```diff
@@ -102,5 +114,4 @@ private static void runTest() {
             exception = e;
         }
     }
-
```

Comment:
I'd rather keep this blank line. Most classes have blank line between the closing brace of the last method and the closing brace of the class itself.